plugins/digitalbitbox/digitalbitbox.py

```diff
@@ -4,7 +4,7 @@
 #
 
 try:
-    from electroncash.bitcoin import TYPE_ADDRESS, push_script, var_int, msg_magic, Hash, verify_message, pubkey_from_signature, point_to_ser, public_key_to_p2pkh, EncodeAES, DecodeAES, MyVerifyingKey, int_to_hex
+    from electroncash.bitcoin import TYPE_ADDRESS, push_script, var_int, msg_magic, Hash, verify_message, pubkey_from_signature, point_to_ser, public_key_to_p2pkh, EncodeAES, DecodeAES, MyVerifyingKey, int_to_hex, aes_encrypt_with_iv, aes_decrypt_with_iv, assert_bytes
```

Comment:
Ok man, I refactored `bitcoin.py` in master. Please go ahead and push another commit on top of this one (or force push a commit amendment to this one, whatever you prefer) that:

1. imports the new `EncodeAES_bytes`, `DecodeAES_bytes`, `hmac_oneshot`
2. Gets rid of any extant references to the old `EncodeAES` and `DecodeAES` (these are now renamed to `*_base64`)
3. Any other clean-up you see..

Again thanks so much for this.